internal/server/handlers.go

```diff
@@ -125,21 +125,13 @@ func (a *API) DeleteUser(c *gin.Context, r *api.Resource) error {
 	return access.DeleteIdentity(c, r.ID)
 }
 
+// TODO: remove after deprecation period
```

Comment:
we can probably come up with a rewrite rule that would work for these rather than a deprecation period.

Reply:
I spent 2 days on it, but it requires adding even more complexity to the API migration system to be able to support "merge" migrations (where two endpoints are implemented by a single one). We'd have to filter the migrations by version and apply them. I think the approach in this PR gives users the same benefits (no breaking change, no deprecated routes in the OpenAPI doc), without us having to do that extra work.